src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../src/images/star.svg);
```

Comment:
The file path used in the background-image property seems to be incorrect. Since the CSS file is already in the `src` directory, the path should be relative to the current directory, not including `src` again. It should be `url(images/star.svg)`.